File: beginner/javascript/1011.js
var input = require('fs').readFileSync('stdin', 'utf8');

var values = input.split('\n');

var pi = 3.14159;

var raio = parseFloat(values.shift());

var volume = (4/3.0) * pi * Math.pow(raio,3);

console.log("VOLUME = " + volume.toFixed(3));

File: beginner/java/1011.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);

            scanner.useLocale(Locale.US);

            double pi = 3.14159;
            double raio = scanner.nextDouble();

            double volume = (4/3.0) * pi * Math.pow(raio,3);

            System.out.printf("VOLUME = %.3f%n", volume);

        scanner.close();
    }
}